validator/packaging/validator.toml.example

```diff
@@ -67,6 +67,19 @@ minimum_peer_connectivity = 3
 # The maximum number of peers that will be accepted.
 maximum_peer_connectivity = 10
 
+# The following settings are for tuning the thread allocation in several of the
+# thread pools used by the validator.  These settings should be modified by
```

Comment:
I suggest stronger, simpler wording: 

"Only advanced users should change these settings."
     or
"Do not change these settings unless you are an experienced Sawtooth user."